### ISSUE TITLE
StackOverflowException on Mac

### ISSUE THREAD

**Issue by comtaler** [firebase/quickstart-unity#27]
After updating to 1.1.0, Unity console keeps throwing the following errors:
```
StackOverflowException
UnityEditor.SearchUtility.CheckForKeyWords (System.String searchString, UnityEditor.SearchFilter filter, Int32 quote1, Int32 quote2) (at /Users/builduser/buildslave/unity/build/Editor/Mono/SearchUtility.cs:104)
UnityEditor.SearchUtility.ParseSearchString (System.String searchText, UnityEditor.SearchFilter filter) (at /Users/builduser/buildslave/unity/build/Editor/Mono/SearchUtility.cs:67)
UnityEditor.AssetDatabase.FindAssets (System.String filter, System.String[] searchInFolders) (at /Users/builduser/buildslave/unity/build/Editor/Mono/AssetDatabaseSearching.cs:17)
UnityEditor.AssetDatabase.FindAssets (System.String filter) (at /Users/builduser/buildslave/unity/build/Editor/Mono/AssetDatabaseSearching.cs:11)
Firebase.Editor.XcodeProjectPatcher.FindConfig (Boolean errorOnNoConfig)
Firebase.Editor.XcodeProjectPatcher.ReadConfigInternal (Boolean errorOnNoConfig)
Firebase.Editor.XcodeProjectPatcher.ReadConfig (Boolean errorOnNoConfig)
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for Firebase.Editor.XcodeProjectPatcher
System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor (RuntimeTypeHandle type) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Runtime.CompilerServices/RuntimeHelpers.cs:101)
UnityEditor.EditorAssemblies.ProcessEditorInitializeOnLoad (System.Type type) (at /Users/builduser/buildslave/unity/build/Editor/Mono/EditorAssemblies.cs:130)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoCMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:519)
System.Reflection.MonoCMethod.Invoke (BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:528)
System.Reflection.ConstructorInfo.Invoke (System.Object[] parameters) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/ConstructorInfo.cs:77)
System.Activator.CreateInstance (System.Type type, Boolean nonPublic) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Activator.cs:372)
System.Activator.CreateInstance (System.Type type) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Activator.cs:254)
UnityEditor.AssetPostprocessingInternal.GetAudioProcessorVersions () (at /Users/builduser/buildslave/unity/build/Editor/Mono/AssetPostprocessor.cs:325)
UnityEditor.AssetPostprocessingInternal:GetAudioProcessorVersions()

```

The project builds, but it slows down the Unity Editor a lot.

**Comment by comtaler** [comment]
No comment on this error?

One note about this is that we have symbolic links in our project.

**Comment by stewartmiles** [comment]
Do you have multiple GoogleServices-Info.plist files in your project?

**Comment by comtaler** [comment]
Yes, I have a duplicate GoogleServices-Info.plist file in my project. After removing the duplicate, the error is gone. Maybe it should not throw exception in this case but output a warning message of the duplicate.

**Comment by stewartmiles** [comment]
Yep, thanks for the stack trace, we'll sort out this bug.

**Comment by stewartmiles** [comment]
This should be resolved in version 1.1.1
https://firebase.google.com/support/release-notes/unity#1.1.1
